unittests/test_policies.py

```diff
@@ -104,10 +107,28 @@ def num_failures_stage(runner, stage):
     return len([t for t in stats.failures() if t.failed_stage == stage])
 
 
+def open_json_schema():
+    # Open and store the JSON schema for later validation
+    schema_filename = os.path.join(reframe.INSTALL_PREFIX, 'reframe',
+                                   'schemas', 'report.json')
+    with open(schema_filename) as fp:
+        try:
+            schema = json.loads(fp.read())
+        except json.JSONDecodeError as e:
+            raise ReframeFatalError(
+                f"invalid configuration schema: '{schema_filename}'"
+            ) from e
```

Comment:
You should not do any exception handling here, since you're in a unit test. If it crashes, the test will fail anyway.